src/widget/wtracktableview.cpp

```diff
@@ -892,6 +935,30 @@ void WTrackTableView::setSelectedTracks(const QList<TrackId>& trackIds) {
     }
 }
 
+bool WTrackTableView::setCurrentTrackId(const TrackId& trackId, const int column = 0) {
+    QItemSelectionModel* pSelectionModel = selectionModel();
```

Comment:
This code is unneeded if pTrackModel->getTrackRows(trackId) returns an empty list. Please move it downwards.

Reply:
done.